.github/workflows/deployment_rollback.yml

```diff
@@ -15,19 +15,19 @@ jobs:
   DeployToCandidateSlot:
     runs-on: ubuntu-latest
     env:
-      ALLOWED_USERS: "lucasdze,jeremy-page,JosiahSiegel,snesm,supriyaaddagada"
+      ALLOWED_USERS: ${{ env.ALLOWED_USERS }}
```

Comment:
Listing the members in the workflow itself is more transparent than in GH settings, which are not visible to all committers to this repo.

Reply:
But doing that causes a problem as any changes will require a build which is not a reasonable operation.
Anyone can print out the value of an repository variable.

Reply:
These users list is used to identify if the github.actor is part of that list but at no point the information of who this user is gets printed or displayed.

I am proposing to simply use ${{ env.ALLOWED_USERS }} and have that driven as an abstraction that we can maintain without having to initiate any sort of change in the build process.

This is only referenced in this specific workflow:
``` console
$ find . -not -iwholename '*/.git/*' -type f | xargs -I {} egrep -iHn "allowed_users" {} ;
./.github/workflows/deployment_rollback.yml:18:   ALLOWED_USERS: "lucasdze,jeremy-page,JosiahSiegel,snesm,supriyaaddagada"
./.github/workflows/deployment_rollback.yml:22:    if: ${{ contains(env.ALLOWED_USERS, github.actor) }}
./.github/workflows/deployment_rollback.yml:26:    if: ${{ !contains(env.ALLOWED_USERS, github.actor) }}
(edited)
```

Reply:
Let me know what's your decision in this regard.

Reply:
Moving the list of allowed users into GH Settings variables is not transparent enough for people who want to find someone who can run the action and do not have access to repository settings.  
I'd prefer to code it in the same manner it was done before, just update the list of users to the current team members.

Reply:
@devopsmatt has pointed out to leave this references in plain text to serve as a reference to others in the development team on who is responsible for the deployments rollbacks.
I will update this list with current/active users and provide references on how to reach out and contact them.

Reply:
Updated list with current resources:
**ALLOWED_USERS**: "devopsmatt,emvaldes,scott-aquia,bethbeza"